lawnotation-ui/supabase/templates/invite.html

```diff
@@ -0,0 +1,6 @@
+<h2>You have been invited</h2>
+<p>You have been invited to create a user on {{ .SiteURL }}.</p>
```

Comment:
I would change this line to <p>You have been invited to create an account on {{ .SiteURL }}.</p>